app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(cp: str) -> None:
```

Comment:
The function does not check if the input string contains exactly three elements and starts with 'cp'. According to the checklist, you should ensure that the input string has 'cp' as the first element and contains three elements separated by spaces.